scripts/shared/lib/utils

```diff
@@ -120,12 +116,26 @@ function declare_cidrs() {
     # shellcheck disable=SC2034 # these variables are used elsewhere
     declare -gA cluster_CIDRs service_CIDRs global_CIDRs
 
-    for i in {1..3}; do
-        add_cluster_cidrs $i
+    i=1
+    for cluster in ${clusters[*]}; do
+        add_cluster_cidrs "$i" "$cluster"
+        i=$(("$i"+1))
     done
 }
 
 function declare_kubeconfig() {
     export KUBECONFIG
-    KUBECONFIG=$(echo "${KUBECONFIGS_DIR}"/kind-config-cluster{1..3} | sed 's/ /:/g')
+    KUBECONFIG=$(find "${KUBECONFIGS_DIR}" -type f | tr '\n' ':')
+}
+
+function print_clusters_message() {
+cat << EOM
```

Comment:
```suggestion
    cat << EOM
```

Since it's technically inside the function :)

Reply:
Done